src/main/resources/lib/plot/blockWrapperTrValidationErrorArea.jelly

```diff
@@ -0,0 +1,16 @@
+<?jelly escape-by-default='true'?>
+<j:jelly xmlns:j="jelly:core" xmlns:d="jelly:define" xmlns:f="/lib/form" xmlns:l="/lib/layout">
+    <!-- TODO remove and switch to div after baseline is 2.264 or newer -->
+    <j:choose>
+        <j:when test="${divBasedFormLayout}">
+            <div>
```

Comment:
this is missing it's validation?

This was one of the points I meant that this shouldn't be needed, something is wrong or weirdly laid out if it's needed
```suggestion
            <div class="validation-error-area">
```